textalyze.js

```diff
@@ -9,12 +9,15 @@ let itemCounts = require('./lib/itemCounts');
 let readFileSync = require('./lib/readFileSync');
 let sanitize = require('./lib/sanitize');
 let stringToChars = require('./lib/stringToChars');
+let process = require('process');
+let file = process.argv[2];
 
 function textalyze(file) {
   return itemCounts(stringToChars(sanitize(readFileSync(file))));
 }
 
-console.log(textalyze('./sample_data/great-gatsby.txt'));
+//console.log(textalyze('./sample_data/great-gatsby.txt'));
+console.log(textalyze(file));
```

Comment:
Write a `dispayItemCounts` function that displays the information in a more sensible way.